modules/media/README.md

```diff
@@ -1,51 +1,58 @@
-## Media Module
+# Media
 
-### 📄 Overview
+## Purpose
 
-Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
-Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+Media module allows users to **upload**, **search** and **edit** media files 
+associated with a specific candidate timepoint in Loris. Any kind of data 
+associated with a candidate timepoint can be uploaded through this module: 
+PDFs, videos, recordings, scripts, log files, etc. Files can optionally be 
+associated to a specific instrument form within a given candidate timepoint.
 
+## Intended Users
 
->Note: Currently editing functionality only allows editing of certain metadata fields, such as `Comments` and `Date of Administration`.
+The Media module is used by data entry staff to upload candidate and timepoint 
+specific media. The module can also be used by data-monitoring staff ensuring the 
+integrity of the data for each candidate.
 
-### 🔒 Permissions
+## Scope
 
-In order to use the media module the user needs one or both of the following permissions:
+The Media module provides a tool to upload files to the server and track uploaded 
+files. Data uploaded must be specific to a candidate and a timepoint and optionally 
+an instrument. The *Edit* functionality only allows modification of the date of
+upload and comments linked to the upload. The *Delete* functionality only hides the 
+file from the front-end without removing the database entry for it nor the file on 
```

Comment:
"The Delete functionality only hides the file on the front-end; it does not remove its database entry nor the file on the server."